runner/runners/polling.go

```diff
@@ -48,6 +48,10 @@ func (r *PollingStatusQuerier) Query(q runner.Query, wait runner.Wait) ([]runner
 			return st, service, err
 		}
 		time.Sleep(r.period)
+
+		// Exponentially increase time between polls
```

Comment:
Suggest adding a max duration that we can just hardcode as a const in this file, (e.g. 10 sec?)